template_feed/Microsoft.DotNet.Common.ItemTemplates/content/Solution/Solution1.sln

```diff
@@ -1,8 +1,8 @@
-﻿
+
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.6.30114.105
-MinimumVisualStudioVersion = 10.0.40219.1
+# Visual Studio Version 15
+VisualStudioVersion = 15.0.26124.0
```

Comment:
Is there a reason you went to 15 for VS Version rather than just fixing the 16 number to be 16.0.30114.105 which is the correct value for 16.6?

Reply:
Thanks for the suggestion! 
I have just updated with the version in this commit https://github.com/dotnet/templating/pull/3007/commits/a971eee1f63af668b7d83bec27ebf01aaf0c3575